### ISSUE TITLE
Investigate state of OLO and RCO in community operatorhub.io

### ISSUE THREAD

**Issue by mingcyu** [WASdev/websphere-liberty-operator#362]
In the initial days of Operators, operatorhub.io was the only place to publish/get Operators. Hence, OLO and RCO were published there. OpenShift UI also included this catalog out of the box. Later on, Red Hat rolled out its Certified Operators Catalog. We completed the certification process to publish OLO and RCO to the Certified Catalog. After some time, Red Hat started its own Community Operators Catalog (most of the Operators were cloned from operatorhub.io) and created their own process. At this time, operatorhub.io also changed its release process. After this, OLO and RCO in operatorhub.io haven't been updated.

Publishing to Red Hat's Certified Catalog has the widest reach as Operators are included in the OperatorHub UI in OpenShift as 'certified'.

operatorhub.io Catalog is only included with the **community** Operator Lifecycle Manager (OLM). Note that OpenShift has its own packaging of OLM that doesn't include the operatorhub.io.

An increasing number of Operators are no longer maintained in operatorhub.io. Instead, they are only maintained in Red Hat's Community Catalog. Red Hat's guidelines specify to **not** publish 'certified' operators to Community Catalog. So we shouldn't start to publish to it.

Investigate whether the effort to publish/maintain OLO and RCO operatorhub.io is justifiable with its reach.
If not, remove OLO and RCO from operatorhub.io as the versions are outdated.
If yes, follow the new release process. This process is different from publishing to Red Hat's Certified Operators Catalog. Operator artifacts _might_ have to be produced differently and maintained.

**Comment by leochr** [comment]
Both RCO and OLO have been removed from operatorhub.io

We decided to remove OLO and RCO from operatorhub.io for the following reasons:
- operatorhub.io is no longer strategic. More and more operators are not published/maintained there. It requires a separate publication process, so probably not worth the effort. We can re-evaluate if that changes in the future
- RCO and OLO are published to more strategic Red Hat's Certified Operators Catalog
- RCO and OLO can be installed standalone (kubectl and customize)
